index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <!-- linking fonts -->
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <!-- lnking  -->
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="navbar">
+            <p class="logo">my logo</p>
+            <img class="burger" src="graphics/burger-menu.svg" alt="">
+            <!-- <nav>
+                <ul>
+                    <li><a href="">portfolio</a></li>
+                    <li><a href="">about</a></li>
+                    <li><a href="">contact</a></li>
+                </ul>
+            </nav> -->
+        </div>
+        <div class="intro">
```

Comment:
- [ ] Please end the `header` tag before `intro` class.
- [ ] Also instead of using `div` tag in the `intro` class use `section` tag.